packages/picker/src/Picker.ts

```diff
@@ -60,6 +60,13 @@ const chevronClass = {
 export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
     protected isMobile = new MatchMediaController(this, IS_MOBILE);
 
+    @property({ type: String })
+    public description = 'sp-picker';
```

Comment:
I'm not sure we need to cary this as a property being we're leveraging a static value in 429, below.

Reply:
I used it as a property because action-menu is leveraging this by inheriting it from picker.
If we declare description merely as a static value then I believe action-menu won't be able to inherit it and leverage it

Reply:
Could you expand on why you want this value settable from the outside?

Reply:
I don't want this value to be settable from outside.
Shall I do like this

```
protected _description= 'sp-picker';

  static get description(): string {
    return this._description;
  }
```

Reply:
This works, you can also hand it directly on the class for a little less code.
```
protected description = 'sp-picker';
```

Historically, the library has simply leveraged these values as magic strings, which would be acceptable here. This means we "know" via convention that the "description" of an element will be id'd as `description` rather than marking that out in sharable code.

However, we do have [this issue](https://github.com/adobe/spectrum-web-components/issues/2064) open to revisit that.

As a "constant", it may be of use to create and export one from this file, so it's not fixedly related to the class:
```
export const DESCRIPTION_ID = 'description';
```

Related, I wanted to work out how we hold that value before questioning the value itself, but the element is _sometimes_ called `sp-picker`, I'm not sure the description should be id'd as such...something a little more clear to what it does would be nice.